.travis.yml

```diff
@@ -1,7 +1,18 @@
----
 language: node_js
+cache:
+  yarn: true
+  directories:
+  - node_modules
 node_js:
-  - stable
+  - '6'
+env:
+  global:
+    - PATH=$HOME/.yarn/bin:$PATH
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+install:
+  - yarn config set workspaces-experimental true
```

Comment:
You can check in a `.yarnrc` file so this is enabled for everyone cloning the repo and so you can remove this step: https://github.com/glimmerjs/glimmer.js/blob/master/.yarnrc

Reply:
I vaguely thought this isn’t needed anymore anyways. Isn’t it set as the default as of yarn 1.0.0?

Reply:
@rwjblue Oh shoot, I think you're right. I think it's opt-out now instead of opt-in.